templates/common/infra/terraform/core/gateway/apim-api/apim-api_output.tf

```
@@ -0,0 +1,3 @@
+output "SERVICE_API_URI" {
+  value = "${data.azurerm_api_management.apim.gateway_url}/${var.api_path}"
+}
```

other .tf files contains line ending.  Do we need it? 
```suggestion
}

```